app/views/users/passwords/edit.html.erb

```diff
@@ -1,4 +1,5 @@
-<h2>Change your password</h2>
+<% @pageTitle = "Set new Password | CAT Forum" %>
+<% @PageDescription = "Set new password for your account" %><h2>Change your password</h2>
 
```

Comment:
Please fix the indentation. Now that many people are working on same
project it is important that we try to keep our code clean.
On 27-May-2015 11:04 pm, "Ananya Maiti" <notifications@github.com> wrote:

> In app/views/users/passwords/edit.html.erb
> <https://github.com/sonalkr132/cat-forum/pull/2#discussion_r31159134>:
>
> > @@ -1,4 +1,5 @@
> > -<h2>Change your password</h2>
> > +<% @pageTitle = "Set new Password | CAT Forum" %>
> > +<% @PageDescription = "Set new password for your account" %><h2>Change your password</h2>
> >
>
> please see beside <% @PageDescription = "Set new password for your
> account" %> in line 2.the
> tag is after the erb tag.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/sonalkr132/cat-forum/pull/2/files#r31159134>.
>